### PR TITLE
First pass at cleaning up RNOTZ cards

### PR DESCRIPTION
**Pull request by zzorba**
I think I got most things right, but not sure the right way to test it out.

A couple questions:
- let me know if I should use a different tag for italics on flavor text.
- On cards with a different illustrator on the front/back, which name do we use?  For locations the front is often generic and the back is more specific?
-  For cards that have a different encounter_code on front/back (new locations in masks, arkham woods locations), which encounter_code/position should they have?  Return or their original one?

---
This change is [Reviewable](https://reviewable.io/reviews/kamalisk/arkhamdb-json-data/297)

### CODE REVIEW

**Comment by drawntotheflame** [comment]
Hey @zzorba,
Which cards have italicised flavour text? 
You can do "back_illustrator" as a tag as well as "illustrator", so both!
I think we should go with the new encounter codes and positions for the new cards.

Just worth noting: we order the tags alphabetically normally so that they're straightforward to check. You don't have to change that this time but in future, for tidiness' sake, please put things in alphabetical order. Thank you!

File: pack/return/rtnotz_encounter.json [hunk 1]
```diff
@@ -220,60 +259,93 @@
...
+        "back_text": "&mdash;Choose one of the two Easttown locations, one of the two Northside locations, one of the two Rivertown locations, and one of the two Miskatonic University locations at random. Put those locations into play. Remove the other versions of Easttown, Northside, Rivertown, and Miskatonic University from the game.<br/>&mdash;When creating the \"Cultist deck,\" include the 3 new <i>Cult of Um&ocirc;rdhoth</i> cards included in this expansion. Shuffle the Cultist deck and remove 3 random cards from it, without looking at them. (<i>You should be left with a 5-card Cultist deck.</i>)<br/>&mdash;If you are instructed to search the encounter deck for an Acolyte, search the encounter deck for any 1-health [[Cultist]] enemy, instead.",
```
Comment (drawntotheflame) [inline pack/return/rtnotz_encounter.json]: For dashes, just space dash space is fine - <- like that!

File: pack/return/rtnotz_encounter.json [hunk 2]
```diff
@@ -316,33 +396,36 @@
...
         "code": "50031",
         "encounter_code": "return_to_the_midnight_masks",
+        "encounter_position": 7,
         "faction_code": "mythos",
         "illustrator": "Guillaume Ducos",
         "name": "Masked Horrors",
         "pack_code": "rtnotz",
         "position": 31,
         "quantity": 2,
+        "flavor": "\"Um&ocirc;rdhoth…Um&ocirc;rdhoth…Um&ocirc;rdhoth…\"",
```
Comment (drawntotheflame) [inline pack/return/rtnotz_encounter.json]: This can be tidied up, like Narogath...

**Comment by drawntotheflame** [comment]
On italics - yes, that's a tricky one. @Kamalisk do you know what should be done here? the new scenario set up cards have text with italics. I suspect a normal italic mark would work ok?

**Comment by Kamalisk** [comment]
just <i> . it is still bold but will be just italic once I make that change live.

**Pull request merged**